Content.Server/GameObjects/Components/Mobs/Speech/owoAccentComponent.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Content.Server.GameObjects.Components.Mobs.Speech
+{
+    [RegisterComponent]
+    public class OwOAccentComponent : Component, IAccentComponent
+    {
+        public override string Name => "OwOAccent";
+
+        private static readonly IReadOnlyList<string> Faces = new List<string>{
+            " (・`ω´・)", " ;;w;;", " owo", " UwU", " >w<", " ^w^"
+        }.AsReadOnly();
+        private string RandomFace => IoCManager.Resolve<IRobustRandom>().Pick(Faces);
+
+        private static readonly Dictionary<string, string> SpecialWords = new Dictionary<string, string>
```

Comment:
This as well could be made more generic as a lot of accents will need this behavior where they replace some words with others.
Also, this shouldn't be static, neither should most of the code be except for regex instances.